apiserver/plane/utils/issue_filters.py

```
@@ -3,8 +3,8 @@
 from datetime import timedelta
 
 from django.utils import timezone
-from django.db.models import Q
-
+from django.db.models import Q, OuterRef
```

Ruff (0.11.9)

`django.db.models.OuterRef` imported but unused

Remove unused import: `django.db.models.OuterRef`

(F401)